app/src/main/java/com/example/aaa/pinkcalculator/Numbers.java

```diff
@@ -1,65 +1,87 @@
 package com.example.aaa.pinkcalculator;
 
+import android.widget.TextView;
+import static com.example.aaa.pinkcalculator.CommonConstants.MINUS;
+import static com.example.aaa.pinkcalculator.CommonConstants.POINT;
+import static com.example.aaa.pinkcalculator.CommonConstants.ZERO;
+
 public class Numbers {
+    private static final String MINUS_AND_NULL = "-0";
+
+    public static boolean canAddNumberToFirstNumberView(MainActivity mainActivity) {
```

Comment:
It's wrong that utils class get from activity something. In your case method should contain MainActivity as parameter

Reply:
Method canAddNumberToFirstNumberView takes MainActivity as parameter.
Or I did't understand "In your case method should contain MainActivity as parameter"?!?
Try to explain what you meant, please.

Reply:
@ilya-shknaj

Reply:
Not good that util class work with activity and get field value from activity. I think for finishing this task it acceptable, but please take it into account.

Reply:
Good, I understood